living_documentation_generator/utils/decorators.py

```diff
@@ -75,7 +75,8 @@ def wrapped(*args, **kwargs) -> Optional[Any]:
                 logger.error("HTTP error calling %s: %s.", method.__name__, e, exc_info=True)
                 return None
             except Exception as e:
-                logger.error("Unexpected error calling %s: %s.", method.__name__, e, exc_info=True)
+                exception_type = type(e).__name__
+                logger.error("%s by calling %s: %s.", exception_type, method.__name__, e, exc_info=True)
```

Comment:
Why do we have an extra line to get exception_type when the method name is the same way?

Reply:
You are right, changed in commit: